docker-compose.yml

```diff
@@ -65,8 +70,19 @@ services:
     ports:
       - "8080:8080"
     volumes:
-      - "${OTOPS_PATH_DEPLOYMENT_WEBAPP}:/srv/site"
-      - "${OTOPS_PATH_CONFIG_WEBAPP}/nginx.conf:/etc/nginx/conf.d/default.conf"
+      - "webapp:/srv/site"
+      - "./config/webapp/nginx.conf:/etc/nginx/conf.d/default.conf"
+    networks:
+      - ot-platform
+
+  openai:
+    image: ${OTOPS_PROFILE_OPENAI_API_DOCKER_IMAGE_NAME}:${OTOPS_PROFILE_OPENAI_API_DOCKER_IMAGE_TAG}
+    container_name: ot-ai-api
+    ports:
+      - 8081:8080 
+    environment:
+      OPENAI_TOKEN: ${OTOPS_PROFILE_OPENAI_TOKEN}
```

Comment:
I may have missed it, but I think we need an additional _target_ in the Makefile, to collect and provision the OpenAI token

Reply:
I think for a local standalone deployment we can leave it as that, this way, whoever is using this deployment, will implement its own _secrets management_ mechanism.